.github/ISSUE_TEMPLATE/00-bug.md

```diff
@@ -11,9 +11,10 @@ labels: 'type: Bug'
 <!--
 What behavior did you observe when encountering this issue?
 What behavior did you expect to observe?
+Describe any work-arounds you've considered, if any.
```

Comment:
I realize this is just refactoring the previous text and that webster's prefers the hyphenated version of "work-around", but I think most of the technical world uses "workaround" now.  Googling leads to several sources that support the non-hyphenated version, including other dictionaries and technical style guides:

* https://dictionary.cambridge.org/us/dictionary/english/workaround
* https://github.com/MicrosoftDocs/microsoft-style-guide/issues/362
* https://ell.stackexchange.com/a/142693
*